### ISSUE TITLE
[MAS4.2.1] [Programmatic and access -Welcome] The Name property must not contain only space characters.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1746]
**Actual Result:**
The Name property contains space characters for “Design Guidelines, From Azure, Continuous deployment, Channels “links 

**Expected Result:**
The Name property must not contain only space characters for “Design Guidelines, From Azure, Continuous deployment, Channels “links

Note: Same issue repro for below path 
Welcome ->Open bot control->Live chat->Edit ngrok setting ->Emulator setting screen
Please refer the attachment:MAS4.2.1_setting.png

**How To Fix:** Provide a UI Automation Name property that concisely identifies the element.

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Welcome tab screen gets opened
3.Navigate to the How To Build A Bot
4.Navigate to the “Design Guidelines ,From Azure, Continuous deployment ,Channels “
5.Run the accessibility insights for windows tool 
6.Observe the issue

**User Impact:**
users who rely on AT will be affected if programmatically name property defines the space characters so it will misguide the user.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Tools: Accessibility insights for Windows 

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}](url)
![MAS4 2 1_k4d_Welcome](https://user-images.githubusercontent.com/53518994/63322146-5604f700-c340-11e9-85bd-9399c34ebb74.png)
![MAS4 2 1_Seting](https://user-images.githubusercontent.com/53518994/63507972-9e661580-c4f6-11e9-9368-b0e439c0b032.png)

**Comment by corinagum** [comment]
I believe this issue should be closed, as Emulator is no longer being tested as a Windows app

**Comment by Kumar2608** [comment]
As directed by customer, electron app wont support Accessibility Insights for Windows
hence we are closing this bug .
